packages/components/src/base-control/stories/index.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * External dependencies
+ */
+import { boolean, text } from '@storybook/addon-knobs';
+
+/**
+ * Internal dependencies
+ */
+import BaseControl from '../';
+
+export default { title: 'Components|Base-control', component: BaseControl };
```

Comment:
@fernandovbs  Thank you for working on this one. I tested it locally, and it's working as expected. The only thing I'd adjust is the title` Base-control` -> `Base Control`

Thank you!~ 🙏

Reply:
Hi @ItsJonQ, Got that. 
Thank you for your reply. I'll change it now :smile: